src/lib/worker.js

```diff
@@ -218,7 +218,7 @@ var podElection = function(pods) {
   });
 
   //Are we the lucky one?
-  return pods[0].status.podIP == hostIp;
+  return pods.length >= 3 && pods[0].status.podIP == hostIp;
```

Comment:
@ienliven I'd like to merge this... just waiting on the reasoning behind `pods.length >= 3`.